server/websocket.go

```diff
@@ -533,7 +607,7 @@ func (p *Plugin) WebSocketMessageHasBeenPosted(connID, userID string, req *model
 	}
 
 	p.mut.RLock()
-	us := p.sessions[userID]
+	us := p.sessions[connID]
```

Comment:
This is a somewhat notable change. We are moving away from mapping call sessions to user ids but instead start using the more granular connection ids. This will let us support multiple, simultaneous sessions per user (e.g. both web and mobile clients connected)